packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/migrate.ts

```diff
@@ -0,0 +1,21 @@
+import { CONTRACT_LIST } from '../../../lib/contracts'
+import { MigrateContract, MigrationMessageMaker } from '../migration'
+
+// TODO: Pending of migrate function on the contract. Serve this as an example of implementation
+type OCR2MigrateMessage = any
```

Comment:
Let's test this and update this comment.

Also, the migration should be versioned somehow. At least in the message. For example, migration from `v0.1.5` to `v1.0.0` requires an Empty message, but the next migration to `v1.1.0` could require a specific non-empty message.

Not sure if we should support more than just the latest migration though. But at least let's be explicit about the version of the message, even if we declare just one (latest).

```suggestion
// Notice: Msg used to migrate OCR2 from v0.1.5 to v1.0.0
type MsgMigrate_1_0_0 = any
```